Resource.js

```diff
@@ -129,12 +135,20 @@ module.exports = function(app, route, modelName, model) {
               })
             });
             break;
+          case 204:
+            // Convert 204 into 200, to preserve the empty result set.
+            if (req.method !== 'DELETE') {
```

Comment:
@randallknutson Yes it will actually.. which I don't want to happen. I think we should perhaps look into removing `204` responses all together, as they aren't helpful in pretty much any case. Thoughts?